PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -4319,8 +4319,10 @@ INT_PTR CALLBACK Finder::run_dlgProc(UINT message, WPARAM wParam, LPARAM lParam)
 				generic_string uncollapseAll = pNativeSpeaker->getLocalizedStrFromID("finder-uncollapse-all", TEXT("Uncollapse all"));
 				generic_string copyLines = pNativeSpeaker->getLocalizedStrFromID("finder-copy", TEXT("Copy Selected Line(s)"));
 				generic_string copyVerbatim = pNativeSpeaker->getLocalizedStrFromID("finder-copy-verbatim", TEXT("Copy"));
+				copyVerbatim += TEXT("\tCtrl+C");

```

Comment:
Not sure in the other language, but in German it's rather "Strg-C"

Reply:
@donho Do you have screenshot of N++ showing "Strg-C" on, for example _Edit_ of main menu?
I can make `notepad.exe` show it that way, but not Notepad++ (still shows "Ctrl-C") while on same screen notepad.exe shows "Strg-C".

Reply:
I don't have screenshot since I don't use any German system.
But I'm sure @chcg can provide one.